src/http.js

```diff
@@ -49,6 +49,9 @@ Zotero.HTTP = new function() {
 	};
 	this.TimeoutError.prototype = Object.create(Error.prototype);
 	
+	// Max size of requested response to load; triggers 400 ResponseSizeError when exceeded
+	this.httpMaxResponseSize = ( config.has('httpMaxResponseSize') ? config.get('httpMaxResponseSize') : 50 ) * 1024 * 1024;
```

Comment:
Since we're adding it to default.json, it should exist even for existing installations, so we might as well use the same default of 10.

Reply:
done